src/ocrd_network/server_utils.py

```diff
@@ -241,3 +247,25 @@ def validate_first_task_input_file_groups_existence(logger: Logger, mets_path: s
         if group not in available_groups:
             message = f"Input file group '{group}' of the first processor not found: {input_file_grps}"
             raise_http_exception(logger, status.HTTP_422_UNPROCESSABLE_ENTITY, message)
+
+
+def kill_mets_server_zombies(minutes_ago=60) -> list[int]:
```

Comment:
Should return `List[int]`

Reply:
Thx [4862d72](https://github.com/OCR-D/core/pull/1283/commits/4862d72fe6f7149ff4ce97d56ac870837bafddc5)